core/src/main/java/hudson/model/Queue.java

```diff
@@ -1521,7 +1521,12 @@ public Api getApi() {
      * also represents the "primary" sub-task (and as implied by this
      * design, a {@link Task} must have at least one sub-task.)
      * Most of the time, the primary subtask is the only sub task.
+     * 
+     * <p>
+     * Tasks are being exported to API in {@link Item#task}, so every child class
+     * has an {@link ExportedBean} attribute.
      */
+    @ExportedBean
     public interface Task extends ModelObject, SubTask {
```

Comment:
In particular [`inQueueTaskLookupByAPI`](https://github.com/jglick/jenkins/blob/6966f33d3db7e7308a201580c9c842f30f7245c9/test/src/test/java/hudson/model/QueueTest.java#L437) could easily be extended to use [`TestTask`](https://github.com/jglick/jenkins/blob/6966f33d3db7e7308a201580c9c842f30f7245c9/test/src/test/java/hudson/model/QueueTest.java#L471), which is not an `@ExportedBean`.